.gitlab-ci.yml

```diff
@@ -167,6 +161,24 @@ check-runtime-benchmarks:          &test
     - time cargo check --features runtime-benchmarks
     - sccache -s
 
+build-wasm-release:
+  stage:                          build
+  <<:                             *collect-artifacts
+  <<:                             *docker-env
+  <<:                             *compiler_info
+  # Note: We likely only want to do this for tagged releases, hence the 'only:'
+  only:
+    # FIXME remove when ready to merge
+    - /^[0-9]+$/
+    # - /^v[0-9]+\.[0-9]+.*$/        # i.e. v1.0, v2.1rc1
+  script:
+    - cargo install wasm-pack
```

Comment:
`wasm-pack` is installed in the docker container: https://github.com/paritytech/scripts/blob/ded5d56cd9c33ea3263dee0bc26192a2c0129f35/dockerfiles/rust-builder/Dockerfile#L50